plugins/woocommerce/client/blocks/assets/js/shared/redux-stores/canvas-mode/test/index.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import { register, select, dispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store } from '../index';
+
+describe( 'Canvas Mode Store', () => {
+	beforeEach( () => {
+		// Reset the store before each test.
+		if ( ! select( 'woocommerce/canvas-mode' ) ) {
+			register( store );
+		}
+	} );
+
+	describe( 'Initial State', () => {
+		it( 'should initialize with isEditMode as false', () => {
+			const state = select( 'woocommerce/canvas-mode' ).isEditMode();
```

Comment:
To get types, we can import the `store` object and use it.

```suggestion
			const state = select( store ).isEditMode();
```